LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2006-2015 Doctrine Project
+Copyright (c) 2006-2025 Doctrine Project
```

Comment:
Let's drop the end date entirely, so that we don't have to do this every year.

```suggestion
Copyright (c) 2006 Doctrine Project
```

Reply:
This should also apply to the other PRs created for the copyright year